web/docs/auth/AuthPills.css

```
@@ -0,0 +1,13 @@
+.display-inline {
+  display: inline;
+}
+
+.pills {
```

This would be a single `pill`

```suggestion
.pill {
```